docs/articles/documentation/test-api/selecting-page-elements/examples-of-working-with-dom-elements.md

```diff
@@ -34,28 +31,52 @@ test('My test', async t => {
 });
 ```
 
-If you need to access an element property not included in the selector's API, request it explicitly by using the [addCustomDOMProperties](selectors/extending-selectors.md) method.
+> Note that all property getters are asynchronous, so add the `await` keyword.
```

Comment:
we have a note about omitting `await` keyword [below](https://github.com/DevExpress/testcafe/pull/3855/files#diff-9cf5819f164b1cff720d3b165912d40bR77).  Maybe we should merge these two notes.